presentation/build.gradle.kts

```diff
@@ -97,7 +97,7 @@ dependencies {
     implementation("io.coil-kt:coil-compose:1.4.0")
     implementation("androidx.work:work-runtime-ktx:2.8.0")
     implementation("androidx.hilt:hilt-work:1.0.0")
-
+    implementation("com.google.accompanist:accompanist-swiperefresh:0.27.0")
```

Comment:
You don't need a dependency for that anymore, there's a normal SwipeToRefresh composable (or similar) available. Just make sure to use the latest stable compose version (I think `1.4.2` atm)

Reply:
@philipplackner Yes, I discovered that after I implemented it. I have the migration guide here https://google.github.io/accompanist/swiperefresh/